clustering/ejb/WFLY-16345_SFSB_externalization_to_remote_Infinispan_server.adoc

```diff
@@ -0,0 +1,91 @@
+= @Stateful Jakarta Enterprise Bean instance externalization to remote Infinispan cluster
+:author:            Paul Ferraro
+:email:             paul.ferraro@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently, WildFly only supports the use of embedded Infinispan for persisting/replicating @Stateful Jakarta Enterprise Bean instances.
+This implementation cannot fully support cross-site replication, since the inability to invalidate entries across sites means that site failback can result in loss of state.
+This feature will add the ability to persist @Stateful Jakarta Enterprise Bean instances in a remote Infinispan cluster via the HotRod protocol, where cross-site replication is fully supported.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-16345[WFLY-16345]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-14953[WFLY-14953]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
+
+* [ ] QE
```

Comment:
@pferraro is it worth adding at least one trivial test into the clustering TS?

Reply:
There are failover and expiration tests included in the proposed implementation.

Reply:
@pferraro I am not sure if we agreed to try and add one test on OpenShift where we check that basic replication works (e.g. two not clustered WF PODs connected to some Infinispan Server, we take one down, we verify the other node has the data)